components/molecules/dropdown/components/colourDropdown.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Organisms - Color picker
+ */
+
+// React
+import React from 'react'
+import { string, func } from 'prop-types'
+import { Position } from 'components/theme'
+import { colourList } from 'components/theme/variables/colorList'
+import { Dropdown, elementTypes } from 'index'
+const colours = colourList
+export const ColourDropdown = ({ children, onColorSelect, position, ...props }) => {
+  return (
+    <Dropdown
+      {...props}
+      items={colours}
+      onChange={e => {
+        onColorSelect(e)
+      }}
+      position={position ?? Position.Bottom}
+      elementType={elementTypes.Colour}
```

Comment:
@mhsnprvr2 Alpha - focus on the details... its more a test than anything - when I know someone is alpha everything means they are looking deeply at every line and quality increases :)